src/client/app/shared/alert.class.ts

```diff
@@ -4,3 +4,8 @@ export class Alert {
 		public type: string,
 		public msg: string) {}
 }
+
+export class Alerts {
+	list: Alert[] = [];
+	map: Map<number, Alert> = new Map<number, Alert>();
```

Comment:
It was in the original: https://github.com/Asymmetrik/mean2-starter/pull/118/files#diff-0c5068ddc631b166e116c9b74fe82d13L10